test/integration/licence_test.rb

```diff
@@ -142,7 +139,7 @@ class LicenceTest < ActionDispatch::IntegrationTest
           ]
 
           stub_licence_exists(
-            "1071-5-1/00BK",
+            "1071-5-1/00AG",
```

Comment:
Can you explain in the commit message why you had to change these?
I'm guessing it's because the tests previously used incorrect values?